Framework/Algorithms/test/PolarizationEfficiencyCorTest.h

```diff
@@ -486,10 +427,35 @@ class PolarizationEfficiencyCorTest : public CxxTest::TestSuite {
     return names;
   }
 
+  std::shared_ptr<PolarizationEfficiencyCor> createAlgorithm() {
+    auto alg = std::make_shared<PolarizationEfficiencyCor>();
+    alg->setRethrows(true);
+    alg->initialize();
+    return alg;
+  }
+
+  void setAlgorithmProperties(const std::shared_ptr<PolarizationEfficiencyCor> &alg,
+                              const std::string &efficiencyMethod, const std::string &method = "",
+                              const std::string &analysisMethod = "") {
+    alg->setProperty("OutputWorkspace", "out");
+    if (!method.empty()) {
+      alg->setProperty("CorrectionMethod", method);
+    }
+    alg->setProperty("Efficiencies", createEfficiencies(efficiencyMethod));
+    if (!analysisMethod.empty()) {
+      alg->setProperty("PolarizationAnalysis", analysisMethod);
+    }
+  }
+
+  void checkWorkspaceGroupSize(size_t expectedSize) {
+    WorkspaceGroup_sptr out = AnalysisDataService::Instance().retrieveWS<WorkspaceGroup>("out");
+    TS_ASSERT_EQUALS(out->size(), expectedSize);
+  }
+
   MatrixWorkspace_sptr createEfficiencies(std::string const &kind) {
     static std::map<std::string, std::vector<std::string>> const labels = {{"Wildes", {"P1", "P2", "F1", "F2"}},
                                                                            {"Fredrikze", {"Pp", "Ap", "Rho", "Alpha"}}};
-    if (kind == "Wildes" || kind == "Fredrikze") {
+    if (kind == WILDES_METHOD || kind == FREDRIKZE_METHOD) {
```

Comment:
It's really great for maintainability that we're using the constants here. There are quite a few places in these tests where we're using "Wildes" and "Fredrikze", so could we go through and replace all of those that we can with the constants?